projects/packages/videopress/src/client/block-editor/blocks/video/components/player/index.tsx

```diff
@@ -228,13 +240,16 @@ export default function Player( {
 				</div>
 			</ResizableBox>
 
-			{ ( ! RichText.isEmpty( caption ) || isSelected ) && (
+			{ showCaption && ( ! RichText.isEmpty( caption ) || isSelected ) && (
 				<RichText
+					identifier="caption"
 					tagName="figcaption"
-					placeholder={ __( 'Write caption…', 'jetpack-videopress-pkg' ) }
+					aria-label={ __( 'Video caption text', 'jetpack-videopress-pkg' ) }
```

Comment:
:heart:

Reply:
Sorry going to change it in a follow up :-(